operator/controllers/manifest_controller.go

```diff
@@ -185,7 +177,10 @@ func (r *ManifestReconciler) sendJobToInstallChannel(ctx context.Context, logger
 			// so remove finalizer in this case, to process with Manifest deletion
 			return r.finalizeDeletion(ctx, manifestObj)
 		}
-		return r.updateManifestStatus(ctx, manifestObj, v1alpha1.ManifestStateError, err.Error())
+		if err := r.updateManifestStatus(ctx, manifestObj, v1alpha1.ManifestStateError, err.Error()); err != nil {
+			return err
+		}
+		return err
```

Comment:
We should only log error and return an update status error - if applicable. 
Since re-queue is anyways enabled on error and unknown errors (like update status) can be logged. 
The same happens on the lifecycle-manager currently.

Reply:
This is what happens here, Line 180 uses an inline-scoped err that overwrites the outer scope. The returned err is not the one from updateManifestStatus, but the original error